IDEAS/Buildings/Data/Frames/Aluminum.mo

```diff
@@ -0,0 +1,14 @@
+within IDEAS.Buildings.Data.Frames;
+record Aluminum "Aluminum frame"
+  extends IDEAS.Buildings.Data.Interfaces.Frame(
+    U_value=2.8);
+       annotation (Documentation(info="<html>
+<p>Aluminum window frame. U value may vary. </p>
```

Comment:
@jelgerjansen can you reformat the html here too? Same comment as in the other pull request.